pkg/utils/reader.go

```diff
@@ -70,9 +73,8 @@ func InitFileReader() *FileReader {
 }
 
 func randomizeBody(f *gofakeit.Faker, m map[string]interface{}, addts bool) {
-	randNum := fastrand.Uint32n(1_000)
-	// sentenceLen := int(fastrand.Uint32n(25))
-	m["batch"] = fmt.Sprintf("batch-%d", randNum)
```

Comment:
lets keep this a string of the form `fmt.Sprintf("batch-%d", f.Number(1, 1000))`